src/ui/public/field_editor/field_editor.js

```diff
@@ -296,16 +335,29 @@ export class FieldEditor extends PureComponent {
     const { field, fieldTypeFormats, fieldFormatId, fieldFormatParams } = this.state;
     const { fieldFormatEditors } = this.props.helpers;
     const defaultFormat = fieldTypeFormats[0] && fieldTypeFormats[0].resolvedTitle;
+    const label = defaultFormat
```

Comment:
__question:__ likely I missed some related review comment from other folks, so just double checking: previously we had `null` if `defaultFormat` was not defined, now we render `Format` label: is that expected?

Reply:
Actually this is the same. Maybe you missed `<span>` outside?

Reply:
Ugh, my bad, sorry :/